src/identity/components/Address.tsx

```diff
@@ -3,17 +3,23 @@ import { getSlicedAddress } from '../getSlicedAddress';
 import { cn, text } from '../../styles/theme';
 import type { AddressReact } from '../types';
 
-export function Address({ address = null, className }: AddressReact) {
+export function Address({
+  address = null,
+  isSliced = true,
+  className,
+}: AddressReact) {
   const { address: contextAddress } = useIdentityContext();
   if (!contextAddress && !address) {
     throw new Error(
       'Address: an Ethereum address must be provided to the Identity or Address component.',
     );
   }
 
+  const displayAddress = contextAddress ?? address;
```

Comment:
The name `display` makes me think this is something we are showcasing. But is not quite correct, as this is more an `address` we use to then do other things.

Same story in the other file.

So what's actually is this `address`? After all this is the `address` that identifies the user using this component. Or provided address.

What name should we pick here instead of `displayAddress`?

And again whatever we pick will be used in the other file as well.

Reply:
`effectiveAddress` - the address being effectively used by the component.

`currentAddress`- The current address in use, highlighting that this is the active address for the component.

`resolvedAddress` - Final resolved address being used by the component, whether it comes from the context or is provided directly as a prop.

Reply:
I'm going to update to `resolvedAddress`, I believe that it's the most appropriate naming here.